src/helpers/client.ts

```diff
@@ -1,6 +1,8 @@
-import {type SanityClient, useClient} from 'sanity'
+import {type SanityClient, useClient, usePerspective} from 'sanity'
```

Comment:
Usage of `usePerspective` also requires a fairly recent version of the `sanity` package (see also my comment here: https://github.com/sanity-io/orderable-document-list/pull/120/files#r2223052210).